apps/client/src/components/Nav.js

```diff
@@ -21,44 +17,12 @@ const Nav = ({
   showNext,
   showPrev,
 }) => {
-  const {
-    topic: { slug: name },
-  } = useContext(CheckerContext);
-  const { trackEvent } = useMatomo();
-  const { path } = useRouteMatch();
-  const route = routeConfig.find((route) => route.path === path);
-  const category = route.matomoPage || route.name;
-
-  const handleNextClick = (e) => {
-    const action = formEnds
-      ? getslug(nextText.toLowerCase())
-      : "form-volgende-knop";
-
-    trackEvent({
-      category,
-      action,
-      name,
-    });
-
-    if (onGoToNext) onGoToNext(e);
-  };
-
-  const handlePrevClick = (e) => {
-    trackEvent({
-      category,
-      action: "form-vorige-knop",
-      name,
-    });
-
-    if (onGoToPrev) onGoToPrev(e);
-  };
```

Comment:
Be removing this you are now not tracking the Prev button anymore

Reply:
Seems like a valid comment. Please provide resolution @svenjens

Reply:
outdated